__tests__/tooling/index.test.js

```diff
@@ -82,6 +82,69 @@ describe('#url([selected])', () => {
         'https://subdomain.example.com:8080/v3'
       );
     });
+
+    it('should use defaults', () => {
+      expect(
+        new Oas({
+          servers: [{ url: 'https://example.com/{path}', variables: { path: { default: 'path' } } }],
+        }).url()
+      ).toBe('https://example.com/path');
+    });
+
+    it('should use user variables over defaults', () => {
+      expect(
+        new Oas(
+          {
+            servers: [{ url: 'https://{username}.example.com', variables: { username: { default: 'demo' } } }],
+          },
+          { username: 'domh' }
+        ).url()
+      ).toBe('https://domh.example.com');
+    });
+
+    it('should fetch user variables from keys array', () => {
+      expect(
+        new Oas(
+          {
+            servers: [{ url: 'https://{username}.example.com', variables: { username: { default: 'demo' } } }],
+          },
+          { keys: [{ name: 1, username: 'domh' }] }
+        ).url()
+      ).toBe('https://domh.example.com');
+    });
+
+    it('should look for variables in selected server', () => {
+      expect(
+        new Oas({
+          servers: [
+            { url: 'https://{username1}.example.com', variables: { username1: { default: 'demo1' } } },
+            { url: 'https://{username2}.example.com', variables: { username2: { default: 'demo2' } } },
+          ],
+        }).url(1)
+      ).toBe('https://demo2.example.com');
+    });
+
+    it.skip('should fetch user variables from selected app', () => {
```

Comment:
skip?

Reply:
This was already being skipped, I just moved the test case into another `describe` block.